spec/features/create_new_work_spec.rb

```diff
@@ -228,6 +228,21 @@
       user.update(last_work_terms_agreement: Time.zone.now.years_ago(2))
     end
 
+    context 'when user clicks cancel link' do
```

Comment:
Can we revert this test? I think that using a feature spec to test link resolution will unnecessarily slow the test suite. This effectively tests that the browser follows links which I believe is something we can take for granted. If we want to add a test, how about checking that the link exists in the component test. The component tests are  much faster.

Reply:
Ok, removed this feature test and added tests to the component test.